assets/app/views/projects.html

```diff
@@ -33,6 +33,8 @@ <h2 class="project truncate">
                       resource-name="{{project.metadata.name}}"
                       project-name="{{project.metadata.name}}"
                       display-name="{{(project | displayName)}}"
+                      type-display-name="project"
```

Comment:
not sure why this is needed? won't it just display "Project" for now, until you do the option on humanize kind to lower case it